data/map.txt

```diff
@@ -15599,6 +15599,7 @@ system "Kor En'lakfar"
 	fleet "Korath Raid" 1200
 	fleet "Korath Home" 720
 	fleet "Korath Large Raid" 15000
+	fleet "Korath Surveillance" 32000
```

Comment:
I've heard complaints that Korath space is getting crowded with ships, if we're going to add fleets consider reducing another very slightly.  Maybe add 50 to "Korath Raid" above, and similar for other systems.  My internet is being hand-delivered sloths right now though so it's hard to check the others, but a modest increase in period to raids of 30 in systems with lots of these to 70 in systems with few of these ought to do the trick.